cli/lib/kontena/cli/vpn/create_command.rb

```diff
@@ -60,9 +60,10 @@ def execute
     def wait_for_configuration_to_finish(token)
       finished = false
       payload = {cmd: ['/usr/local/bin/ovpn_getclient', 'KONTENA_VPN_CLIENT']}
+      service = client(require_token).get("services/#{current_grid}/vpn/server/containers", payload)['containers'][0]
       until finished
         sleep 3
-        stdout, stderr = client(require_token).post("services/#{current_grid}/vpn/server/containers/1/exec", payload)
+        stdout, stderr = client(require_token).post("containers/#{service['id']}/exec", payload)
```

Comment:
Is this really right? Shouldn't the path be something like `containers/<node>/vpn-server-1/exec`